src/components/Player/GridControls.js

```diff
@@ -20,6 +20,8 @@ export default class GridControls extends Component {
     right: this.setDirectionWithCallback('across', this.moveSelectedBy(0, 1).bind(this)).bind(this),
     forward: this.moveSelectedUsingDirection(1).bind(this),
     backward: this.moveSelectedUsingDirection(-1).bind(this),
+    home: this.moveToEdge(true),
```

Comment:
hmm, this reminds me a bit of ctrl a / ctrl e / ctrl u. that might be interesting too

Reply:
I'll leave this as is for now but if someone else is interested in adding this feature, it shouldn't be too difficult to do.